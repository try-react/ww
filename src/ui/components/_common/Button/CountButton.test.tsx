import React from "react";
import { CountButton } from ".";
import renderer from "react-test-renderer";

it("Snapshot", () => {
  const tree = renderer.create(<CountButton>テスト</CountButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
