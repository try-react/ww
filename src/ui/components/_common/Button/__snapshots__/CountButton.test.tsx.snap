// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<button
  className="count-button"
  type="button"
>
  テスト
</button>
`;
